dts/arm/silabs/efr32bg22.dtsi

```diff
@@ -54,11 +54,31 @@
 		};
 
 		usart0: usart@5005c000 {
-			compatible = "silabs,gecko-usart";
+			compatible = "silabs,gecko-spi-usart";
 			reg = <0x5005C000 0x400>;
 			interrupts = <13 0>, <14 0>;
 			interrupt-names = "rx", "tx";
 			status = "disabled";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			cs-gpios = <&gpioc 3 GPIO_ACTIVE_LOW>;
+
+			status = "disabled";
+			mx25r80: mx25r8035f@0 {
+				compatible = "jedec,spi-nor";
+				reg = <0>;
+				spi-max-frequency = <80000000>;
+				size = <0x800000>;
+				jedec-id = [c2 28 14];
+				sfdp-bfp = [
+					 e5 20 f1 ff  ff ff 7f 00  44 eb 08 6b  08 3b 04 bb
+					 ee ff ff ff  ff ff 00 ff  ff ff 00 ff  0c 20 0f 52
+					 10 d8 00 ff  23 72 f5 00  82 ed 04 b7  44 83 38 44
+					 30 b0 30 b0  f7 c4 d5 5c  00 be 29 ff  f0 d0 ff ff
+				];
```

Comment:
is this flash embedded in the SoC?

Reply:
It is a separate on-board chip, I guess I should move this to board-specific dts then

Reply:
moved `mx25r80` to `boards/arm/efr32bg_sltb010a/efr32bg_sltb010a.dts`